python_modules/dagster/dagster/core/definitions.py

```diff
@@ -458,7 +461,7 @@ def _create_execution_structure(name, solids, dependencies_dict):
     return dependency_structure, pipeline_solid_dict
 
 
-def _validate_dependences(dependencies, solid_dict):
+def _validate_dependencies(dependencies, solid_dict, alias_lookup):
```

Comment:
infelicity!

Reply:
Lol if you mean the programming and not the spellinbg I agree that this is ugly. I don't know a good name here for `alias_lookup` -- I did consider instead either a) inlining the logic of _validate_dependencies or b) constructing some kind of bidict.